pipeline-log/handler_test.go

```diff
@@ -18,3 +19,34 @@ func Test_getPath(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func Test_tlsEnabled(t *testing.T) {
```

Comment:
LGTM, but what about the other flags?

Reply:
Ah I will add tests for all existing s3 flags.

Reply:
Moved bucketName to a function `bucketName() string` and made 1 additional test. I am not sure for `region` though. Have a default value? Exit when region does not exist ?